README.md

```diff
@@ -7,3 +7,28 @@ This is a suitcase subpackage for reading a particular file format.
 ```
 pip install suitcase-tiff
 ```
+
+## Quick Start
+
+`suitcase-tiff` supports stack and series.
+
+Using stack:
+
+```
+import suitcase.tiff_stack
+docs = db[-1].documents(fill=True)
+suitcase.tiff_stack.export(docs, 'my_exported_files/')
+```
+
+Using series:
+
+```
+import suitcase.tiff_series
+docs = db[-1].documents(fill=True)
+suitcase.tiff_series.export(docs, 'my_exported_files/')
+```
+The exported file will have path `my_exported_files/8fe81a89-8103-487a-85d2-23c70af96f16-primary.tiff`. The default file prefix can be changed with the `file_prefix` keyword argument. See documentation link below.
```

Comment:
Can this explain the different paths for series and stack? Also it might be helpful to clairty that it will have a path _like_ this when `8fe...` is the unique ID of the Run.